manifests/init.pp

```diff
@@ -151,7 +151,7 @@
   Optional[String]  $memory_replacement_policy     = $squid::params::memory_replacement_policy,
   Optional[Boolean] $httpd_suppress_version_string = $squid::params::httpd_suppress_version_string,
   Optional[Variant[Enum['on', 'off', 'transparent', 'delete', 'truncate'], Boolean]]
-                    $forwarded_for                 = $squid::params::forwarded_for,
+  $forwarded_for                 = $squid::params::forwarded_for,
```

Comment:
I guess they wanted alignment here. It's why I think alignment is bad

Reply:
fixed it